vllm/model_executor/models/llama_eagle3.py

```diff
@@ -328,6 +328,12 @@ def load_weights(self, weights: Iterable[tuple[str, torch.Tensor]]):
                 includes_embed_tokens = True
             model_weights[name] = loaded_weight
 
+            # To prevent overriding with target model's layers
+            if "lm_head" in name:
+                self.has_own_lm_head = True
+            if "embed_tokens" in name:
```

Comment:
Is this a change from the default? I thought that EAGLE3 heads usually share the embedding with the base model?